cli/generate_operator_keys.go

```diff
@@ -15,6 +18,8 @@ var generateOperatorKeysCmd = &cobra.Command{
 	Use:   "generate-operator-keys",
 	Short: "generates ssv operator keys",
 	Run: func(cmd *cobra.Command, args []string) {
+		encryptionPassword, _ := cmd.Flags().GetString("encryption-password")
```

Comment:
`encryption-password` is long, `password` should be enough.
```suggestion
		encryptionPassword, _ := cmd.Flags().GetString("password")
```